bambi/tests/test_predict.py

```diff
@@ -356,3 +354,25 @@ def test_predict_include_group_specific():
 
     # When we include group-specific terms, these predictions are different
     assert not (idata_1.posterior["y_mean"] == idata_1.posterior["y_mean"][:, :, 0]).all()
+
+
+def test_predict_offset():
```

Comment:
These tests show that the model compiles and MCMC runs. What would be nice is a check that ensures the correct parameters are recovered as well. Not required for this PR just mentioning it